### PR TITLE
add store live templates and readme for live templates

### PR DESCRIPTION
**Pull request by lsaudon**
#188

### CODE REVIEW

File: tool/intellij/Mobx.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<templateSet group="Mobx">
+  <template name="str" value="part '$NAME$.g.dart';&#10;&#10;class $NAMECAPITALIZE$ = _$NAMECAPITALIZE$ with _$$$NAMECAPITALIZE$;&#10;&#10;abstract class _$NAMECAPITALIZE$ with Store {&#10;  @observable&#10;  int value = 0;&#10;&#10;  @action&#10;  void increment() {&#10;    value++;&#10;  }&#10;}" description="Mobx Store" toReformat="true" toShortenFQNames="true">
```
[the remark this answers is not in the document]
Comment (pavanpodila) [reply tool/intellij/Mobx.xml]: Also it would be good to have templates for `@observable` and `@action` 👍

File: tool/intellij/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+## Live Templates for MobX
+
+### Add Live Templates
+
+Copy file `MobX.xml` to `AndroidStudio3.4/templates/`
```
Comment (pavanpodila) [inline tool/intellij/README.md]: - You can keep the version part as `<version>` 
- Also can you include the path to Intellij as well ?

Comment (pavanpodila) [reply tool/intellij/README.md]: Looking good!

File: tool/intellij/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+## Live Templates for MobX
+
+### Add Live Templates
+
+Copy file `MobX.xml` to:
```
Comment (pavanpodila) [inline tool/intellij/README.md]: I think here you can create two sections, one for Android and other for Intellij. I think we have both kinds of users.

Comment (pavanpodila) [reply tool/intellij/README.md]: Rest looks great

Comment (lsaudon) [reply tool/intellij/README.md]: I don't know the paths to Intellij.
You want me to make an AndroidStudio folder and an Intellij folder ?

**Pull request merged**